Lib/test/pyclbr_input.py

```diff
@@ -17,12 +17,7 @@ class C (B):
 
     d = 10
 
-    # XXX: This causes test_pyclbr.py to fail, but only because the
-    #      introspection-based is_method() code in the test can't
-    #      distinguish between this and a genuine method function like m().
-    #      The pyclbr.py module gets this right as it parses the text.
-    #
-    #f = f
+    f = f
```

Comment:
```suggestion
    #f = f
```
I believe you misinterpreted the comment above this line.  With this change, is_method sees f as a method and *erroneously* adds it to 'actualMethods', which makes the test fail because the current code does not add aliases to 'foundMethods'. 
```
>>> import pyclbr as br
>>> t = br.readmodule_ex("pyclbr_input", ["f:/dev/3x/lib/test"])
>>> t['C'].children
{'m': <pyclbr.Function object at 0x000002225FA58D60>, 'sm': <pyclbr.Function object at 0x000002225FA58EB0>, 'cm': <pyclbr.Function object at 0x000002225FA58FD0>}
>>> t['C'].methods
{'m': 22, 'sm': 25, 'cm': 28}
```
I do not see any current code intended to add a Function object for aliases like `f`.  Adding code to fix the test bug you introduce adds a change in behavior.  (I also uncommented the `f=f` line in installed 3.9 and got the same result.)